src/identity/apis/org.ts

```diff
@@ -101,21 +100,20 @@ export const fetchDefaultAccountDefaultOrg = async (): Promise<
   if (response.status === 500) {
     throw new ServerError(response.data.message)
   }
-  const {data} = response
+  const {data: accounts} = response
```

Comment:
:+1: `accounts` is a much better name than `data`.

```ts
const accounts = response.data
```
is an alternative to this destructuring syntax which has less syntax (fewer things like `{` and `:`) and might be easier to follow

Reply:
agreed! I think my way might be unnecessarily fancy. I like your straight-forward approach